scripts/quick_start.sh

```diff
@@ -32,7 +32,11 @@ run_xline() {
     --storage-engine rocksdb \
     --data-dir /usr/local/xline/data-dir \
     --auth-public-key /mnt/public.pem \
-    --auth-private-key /mnt/private.pem"
+    --auth-private-key /mnt/private.pem \
+    --client-listen-urls=http://${SERVERS[$1]}:2379 \
```

Comment:
Should it be 2380？

Reply:
Generally, the outside clients use port 2379, and cluster peers use port 2380 for inter-peer communication